src/patternfly/components/Tabs/tabs.scss

```diff
@@ -112,70 +131,39 @@
   max-width: 100%;
   overflow-x: auto;
   -webkit-overflow-scrolling: touch;
-
-  // Right border on last tab
-  .pf-c-tabs__item:last-of-type .pf-c-tabs__button::before {
-    border-right-width: var(--pf-c-tabs__item--BorderWidth);
-  }
+  transition: .2s;
 }
 
-// Tab items
+// Tabs item
 .pf-c-tabs__item {
+  display: flex;
   flex: none;
-
-  // Tab item hover state
-  &:not(.pf-m-current):hover .pf-c-tabs__button::after {
-    position: absolute;
-    top: 0;
-    right: 0;
-    bottom: 0;
-    left: 0;
-    margin-left: var(--pf-c-tabs__item--BorderWidth);
-    content: "";
-    border-top: var(--pf-c-tabs__item--m-current--BorderTopWidth) solid var(--pf-c-tabs__item--hover--Color);
-  }
-
-  // Tab item current state
-  &.pf-m-current .pf-c-tabs__button {
-    color: var(--pf-c-tabs__item--m-current--Color);
-
-    &::before {
-      border-bottom-color: var(--pf-c-tabs__item--BackgroundColor);
-    }
-
-    &::after {
-      position: absolute;
-      top: 0;
-      right: 0;
-      bottom: 0;
-      left: 0;
-      margin-left: var(--pf-c-tabs__item--BorderWidth);
-      content: "";
-      border-top: var(--pf-c-tabs__item--m-current--BorderTopWidth) solid var(--pf-c-tabs__item--m-current--Color);
+  align-items: stretch;
+
+  // Current
+  &.pf-m-current {
+    .pf-c-tabs__button,
+    .pf-c-tabs__button:hover::after {
+      --pf-c-tabs__button--Color: var(--pf-c-tabs__item--m-current--Color);
+      --pf-c-tabs__button--after--BorderColor: var(--pf-c-tabs__item--m-current--c-tabs__button--after--BorderColor);
     }
   }
 
-  // Secondary tab states
-  .pf-m-tabs-secondary & .pf-c-tabs__button {
-    // Remove current state and hover state
-    &::before,
-    &::after,
-    &:hover::after {
-      content: none;
-    }
+  &:not(:first-child) .pf-c-tabs__button::after {
+    margin-left: var(--pf-c-tabs__item--BorderWidth);
```

Comment:
This is creating a gap that I'm assuming is on purpose, but it's kind of odd looking with the bottom border showing through between the gap. Why do we need the gap?

<img width="125" alt="Screen Shot 2020-03-04 at 7 25 53 PM" src="https://user-images.githubusercontent.com/35148959/75938806-436e8d00-5e4e-11ea-8a33-9b18b19d9fde.png">